src/main/java/org/jenkinsci/bytecode/AdaptField.java

```diff
@@ -62,10 +62,12 @@ void parse(TransformationSpec spec, AnnotatedElement e) {
             if (name.length()==0)   name = mem.getName(); // default to the same name
 
             MemberAdapter mrs = null;
-            if (e instanceof Field)
+            if (e instanceof Field) {
                 mrs = fieldToField((Field) e);
-            if (e instanceof Method)
+            }
+            if (e instanceof Method) {
```

Comment:
yes but it helped when debugging and we do not need to worry about merge conflicts and back-porting here as this is a linear release train (yay for almost CD)